### ISSUE TITLE
VS Code doesn't show warning even though pub command does.

### ISSUE THREAD

**Issue by chooyan-eng** [kawa1214/import-lint#9]
Hi, thank you for publishing this awesome package!

I've just started to try this package and I found my VS Code doesn't show any warnings, even if `flutter pub run import_lint` command does.

This problem also happens on `example` project of this package.

What I did was below.

- `git clone` the repo of `import-lint`
- open `example` directory on VS Code
- run `flutter pub get`
- run `flutter pub run import_lint`
- open `lib/repository/test_two_repository.dart` file.

and my `flutter doctor` is below.

```
$ flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel master, 2.13.0-0.0.pre.390, on macOS 12.0.1 21A559 darwin-arm, locale ja-JP)
[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 13.3)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2020.3)
[✓] VS Code (version 1.61.2)
[✓] Connected device (2 available)
    ! Error: Failed to prepare device for development. Please unlock and reconnect the device. (code
      806)
[✓] HTTP Host Availability

• No issues found!
```

Do I need to do anything in addition to show up warnings on the editor?

Thanks.

**Comment by chooyan-eng** [comment]
I found my VS Code output the crash log below. Hope it's related to this issue and helps you solving it. 

```
### Versions

- Visual Studio Code v1.61.2
- Dart Code v3.29.0
- Flutter SDK v2.10.4

### Analyzer Info

The analyzer was launched using the arguments:

- language-server
- --protocol=analyzer
- --client-id=VS-Code
- --client-version=3.29.0

### Exception

Unexpected error: RangeError (end): Invalid value: Not in inclusive range 1..100: 104

#0      RangeError.checkValidRange (dart:core/errors.dart:355:9)
#1      _StringBase.substring (dart:core-patch/string_patch.dart:400:27)
#2      new ImportLintAnalyze.ofFile (package:import_lint/src/import_lint_analyze.dart:27:39)
#3      ImportLintPlugin._checkFile (package:import_lint/plugin/plugin.dart:172:27)
#4      ImportLintPlugin.createAnalysisDriver.<anonymous closure>.<anonymous closure> (package:import_lint/plugin/plugin.dart:89:28)
#5      _rootRunUnary (dart:async/zone.dart:1442:13)
#6      _CustomZone.runUnary (dart:async/zone.dart:1335:19)
#7      _CustomZone.runUnaryGuarded (dart:async/zone.dart:1244:7)
#8      _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:341:11)
#9      _BufferingStreamSubscription._add (dart:async/stream_impl.dart:271:7)
#10     _SyncBroadcastStreamController._sendData.<anonymous closure> (dart:async/broadcast_stream_controller.dart:385:20)
#11     _BroadcastStreamController._forEachListener (dart:async/broadcast_stream_controller.dart:322:15)
#12     _SyncBroadcastStreamController._sendData (dart:async/broadcast_stream_controller.dart:384:5)
#13     _BroadcastStreamController.add (dart:async/broadcast_stream_controller.dart:244:5)
#14     _AsBroadcastStreamController.add (dart:async/broadcast_stream_controller.dart:474:11)
#15     _RootZone.runUnaryGuarded (dart:async/zone.dart:1618:10)
#16     _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:341:11)
#17     _DelayedData.perform (dart:async/stream_impl.dart:591:14)
#18     _StreamImplEvents.handleNext (dart:async/stream_impl.dart:706:11)
#19     _PendingEvents.schedule.<anonymous closure> (dart:async/stream_impl.dart:663:7)
#20     _microtaskLoop (dart:async/schedule_microtask.dart:40:21)
#21     _startMicrotaskLoop (dart:async/schedule_microtask.dart:49:5)
#22     _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:122:13)
#23     _Timer._runTimers (dart:isolate-patch/timer_impl.dart:402:11)
#24     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:192:12)
```

Thanks.

**Comment by kawa1214** [comment]
@chooyan-eng

Thank you for the report and log!
I will fix this error.

**Comment by kawa1214** [comment]
@chooyan-eng 
Thank you for providing the error log.i may be able to identify the cause.

However, the example project worked in my environment.
Can you provide me code and analysis_options.yaml?
